IdentityCore/src/util/NSDictionary+MSIDExtensions.m

```diff
@@ -96,54 +96,46 @@ - (NSDictionary *)dictionaryByRemovingFields:(NSArray *)fieldsToRemove
     return mutableDict;
 }
 
+- (BOOL)msidAssertType:(Class)type ofKey:(NSString *)key required:(BOOL)required error:(NSError **)error
+{
+    return [self msidAssertTypeIsOneOf:@[type] ofKey:key required:required error:error];
+}
 
-- (BOOL)msidAssertType:(Class)type
-               ofField:(NSString *)field
-               context:(id <MSIDRequestContext>)context
-             errorCode:(NSInteger)errorCode
-                 error:(NSError **)error
+- (BOOL)msidAssertTypeIsOneOf:(NSArray<Class> *)types ofKey:(NSString *)key required:(BOOL)required error:(NSError **)error
```

Comment:
should we add a unit test for this method?

Reply:
done